cvat-ui/src/reducers/index.ts

```diff
@@ -124,20 +117,51 @@ export interface TasksState {
 
 export interface ExportState {
     tasks: {
-        [tid: number]: string[];
+        [tid: number]: {
+            'dataset': string[],
+            'backup': boolean,
```

Comment:
```suggestion
            'dataset': string[];
            'backup': boolean;
```

The same for others below